rest-api-spec/src/main/resources/rest-api-spec/api/cat.cluster_manager.json

```diff
@@ -0,0 +1,51 @@
+{
+  "cat.cluster_manager":{
+    "documentation":{
+      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-master/",
```

Comment:
Should we create a task to update our website, and subsequently this entry?

Reply:
You got a good point! Thank you. I will mention this explicitly in the issue I created in yesterday https://github.com/opensearch-project/documentation-website/issues/450

Reply:
In addition, guess what, the current documentation url for all the REST API specs point to Elastic websites. 😄